### PR TITLE
feat(squash-commit): update formik into react hook form

### PR DESCRIPTION
**Pull request by ankush29**
squash commit

commit ca4d5062ecf8d5be4757649e90814cb32bb1ff78
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Sat Mar 28 00:24:17 2020 +0530

    feat(update-test): update test

    update test for form

commit c0fa077c8b6612ec52ace5c9f4561483480974e5
Merge: 3eddc97 54e563c
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Sat Mar 28 00:05:34 2020 +0530

    Merge branch 'develop' of https://github.com/pagesource/atomic-react-components into develop

commit 3eddc97030b3dba6c4941b5603f1b4cd93fb202e
Merge: a0fb264 ec497d0
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Fri Mar 27 23:48:39 2020 +0530

    update test case

commit a0fb2647a15d26b84c01d1c46a7f3808163dfa31
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Thu Mar 26 15:23:44 2020 +0530

    update file input

commit 8b74692e574268359793c388b5d5c41d7b5aa900
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Wed Mar 25 22:43:12 2020 +0530

    update file tag

commit 468b8989362525d93e0f1cd9617b17f9c3601607
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Wed Mar 25 22:38:55 2020 +0530

    update file tag

commit 166ad7770758617eb552ba2e8ce760202bc6db70
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Wed Mar 25 22:36:40 2020 +0530

    update file tag

commit b523c0dad0cc897fe4186b28d5bd046034b93afb
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Wed Mar 25 20:39:04 2020 +0530

    update toggle

commit 545e317a7cc5170b051075f9c3f002350aa5bdd6
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Wed Mar 25 19:59:52 2020 +0530

    update checkbox and textarea

commit 45c90fca29883933b6f1d030e75cd13e22ad43e9
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Wed Mar 25 16:52:48 2020 +0530

    update iradio

commit 96ae4dfad41d4e32d5f61133b898761334ed9adb
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Wed Mar 25 16:22:20 2020 +0530

    update iradio

commit 9cf565889c5d6d8b901d1f04a4c289671e8a481a
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Wed Mar 25 15:19:59 2020 +0530

    update select

commit 01c1318217a36350846f28865546c646a034d075
Author: Ankush Saini <anksaini1@publicisgroupe.net>
Date:   Wed Mar 25 00:19:31 2020 +0530

    fix(formik-to-react-hook-form): formik to react hook form

    update input type only

### CODE REVIEW

File: lib/components/atoms/Input/Input.js [hunk 1]
```diff
@@ -20,20 +20,22 @@ const Input = React.forwardRef(
...
+      ref,
+      'aria-label': ariaLabel,
+      className,
+      name,
+      type,
+      placeholder,
+      'data-locator': elementLocator || `input-${type}-${name}`,
+      ...others,
+    };
+    if (type === 'radio') propArray.value = inputValue;
```
Comment (vinodloha) [inline lib/components/atoms/Input/Input.js]: Shouldn't this check be type !== 'file'

File: lib/components/atoms/Input/Input.js [hunk 2]
```diff
@@ -20,20 +20,22 @@ const Input = React.forwardRef(
...
+    const propArray = {
+      id,
+      ref,
+      'aria-label': ariaLabel,
+      className,
+      name,
+      type,
+      placeholder,
+      'data-locator': elementLocator || `input-${type}-${name}`,
+      ...others,
```
Comment (vinodloha) [inline lib/components/atoms/Input/Input.js:33]: Also others should be placed first and explicit props should be more weight

File: lib/components/molecules/Form/DragDropInput/DragDropInput.js [hunk 3]
```diff
@@ -24,45 +23,40 @@ const DDFieldInput = ({
...
   localeText,
+  register,
+  setValue,
   ...others
 }: DragDropInputProps): Node => {
   const fileInput = React.createRef();
   const { onDragOver, onDragLeave, onDrop, onFilesAdded, getHighlightClass } = useDragAndDrop();
   const styleCls = getHighlightClass(className);
+  useEffect(() => {
+    console.log('name----', { name });
```
Comment (vinodloha) [inline lib/components/molecules/Form/DragDropInput/DragDropInput.js]: Please clean this log

File: lib/components/atoms/Textarea/tests/__snapshots__/Textarea.style.test.js.snap [hunk 4]
```diff
@@ -1,20 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Select Styled component should render correctly 1`] = `
-.c0 {
```
Comment (vinodloha) [inline lib/components/atoms/Textarea/tests/__snapshots__/Textarea.style.test.js.snap:4]: If we want we can remove the style testing, we are not Doing it much anyway.

File: lib/components/atoms/Input/Input.js (hunk 1 above)
Comment (ankush29) [reply lib/components/atoms/Input/Input.js]: we need value props only for radio type

Comment (ankush29) [reply lib/components/atoms/Input/Input.js]: fixed

File: lib/components/molecules/Form/DragDropInput/DragDropInput.js (hunk 3 above)
Comment (ankush29) [reply lib/components/molecules/Form/DragDropInput/DragDropInput.js]: done

**Pull request closed without merging**